src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,8 +1150,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
```

Comment:
Good job making the parameters more readable.
However does the method really remove prefix at the start of the string? 
Or does it just remove all instances of based on String prefix?
Is this method implemented based on the suggested general method?